modules/ROOT/pages/deployment/general/general-info.adoc

```diff
@@ -304,6 +304,15 @@ NOTE: You must set this environment variable to a valid path when using the manu
 
 WARNING: The location must exclusively be used by Infinite Scale. Writing into this location not using Infinite Scale is strictly discouraged to avoid any unexpected behaviour. 
 
+[IMPORTANT]
+====
+Consider using an own partition or an external filesystem like NFS for the data path. If you only have one partition for your OS, Infinite Scale and your data, *filling up the filesystem with user data can make your system unresponsible*. This can easily happen with following conditions: 
```

Comment:
```suggestion
Consider using a separate partition or an external filesystem like NFS for the data path. If you only have one partition for your OS, Infinite Scale and your data, *filling up the filesystem with user data can make your system unresponsive*. This can easily happen under the following conditions: 
```